packages/test/src/load/starter.ts

```diff
@@ -4,9 +4,23 @@ import { bufferTime, map, mergeMap, tap, takeUntil } from 'rxjs/operators';
 import { Connection, WorkflowClient } from '@temporalio/client';
 import { StarterArgSpec, starterArgSpec, getRequired } from './args';
 import * as os from 'os';
+import { smorgasbord } from '../workflows';
 
 async function runWorkflow(client: WorkflowClient, name: string, taskQueue: string) {
-  await client.execute(name, { taskQueue });
+  if (name === 'smorgasbord') {
```

Comment:
This can probably be done in an activity started by the workflow, I'd rather keep the starter generic if possible.

Reply:
done